src/components/common/Members/MembersContent.tsx

```diff
@@ -27,7 +28,7 @@ const MembersContent = ({
   contributors,
 }: Props) => {
   const isRootDomain =
-    selectedDomain === ROOT_DOMAIN_ID ||
+    selectedDomain === Id.RootDomain ||
     selectedDomain === COLONY_TOTAL_BALANCE_DOMAIN_ID;
```

Comment:
Same comment as I left above. It's either root, or all domains, can't really be both (unless we're overloading)